src/build/build_step.go

```diff
@@ -486,7 +486,7 @@ func runBuildCommand(state *core.BuildState, target *core.BuildTarget, command s
 	}
 	env := core.StampedBuildEnvironment(state, target, inputHash, path.Join(core.RepoRoot, target.TmpDir()), target.Stamp)
 	log.Debug("Building target %s\nENVIRONMENT:\n%s\n%s", target.Label, env, command)
-	out, combined, err := state.ProcessExecutor.ExecWithTimeoutShell(target, target.TmpDir(), env, target.BuildTimeout, state.ShowAllOutput, process.NewSandboxConfig(target.Sandbox, target.Sandbox), command)
+	out, combined, err := state.ProcessExecutor.ExecWithTimeoutShell(target, target.TmpDir(), env, target.BuildTimeout, state.ShowAllOutput, false, process.NewSandboxConfig(target.Sandbox, target.Sandbox), command)
```

Comment:
Don't know if this is really an issue but would it better for readability to say
```
foreground := false
out, combined, err := state.ProcessExecutor.ExecWithTimeoutShell(..., foreground, ...)
```
?

Reply:
I don't think it's worth it. Jumping quickly to the function tells you exactly what that parameter is about. Also, if I were to make this change here, I would have to create the same variable in all other places where this function call exists.